packages/rest-hooks/src/__tests__/restClient.test.ts

```diff
@@ -0,0 +1,65 @@
+import { jest } from '@jest/globals' // Import jest from @jest/globals
+import axios from 'axios'
+import { createRestClient, _expandRestPath } from '../restClient'
+
+// Mock axios.create to prevent actual network requests
```

Comment:
So what is the connection of axios to the module of our testing? Comments should add that context but I think that the developer could easily gather that information once he comes in and reads the module code. So if it was me, I would remove it. But I will leave it to you how you will tackle this.

Reply:
Remove this also